src/solvers/dg_curved/dg.jl

```diff
@@ -16,6 +16,32 @@ end
 end
 
 
+@inline function calc_boundary_flux_by_direction!(surface_flux_values::AbstractArray{<:Any,4}, u, t, orientation,
+                                          boundary_condition::BoundaryConditionPeriodic, equations, mesh::CurvedMesh, 
+                                          dg::DG, cache, direction, node_indices, surface_node_indices, element)
```

Comment:
```suggestion
@inline function calc_boundary_flux_by_direction!(surface_flux_values::AbstractArray{<:Any,4}, u, t, orientation,
                                                  boundary_condition::BoundaryConditionPeriodic, equations, mesh::CurvedMesh, 
                                                  dg::DG, cache, direction, node_indices, surface_node_indices, element)
```

Reply:
Ah, this damn indentation breaks every time.